src/components/Facility/FacilityUsers.tsx

```diff
@@ -71,10 +71,15 @@ export default function FacilityUsers(props: { facilityId: string }) {
       title={t("users_management")}
       componentRight={
         <Badge
-          className="bg-purple-50 text-purple-700 ml-2 text-sm font-medium rounded-xl px-3 m-3"
+          className="bg-purple-50 text-purple-700 ml-2 text-sm font-medium rounded-xl px-3 m-3 w-max"
           variant="outline"
         >
-          {t("user_count", { count: userListData?.count ?? 0 })}
+          {userListLoading
```

Comment:
Switch from isLoading to isFetching for useQuery in all 5 files; isLoading only corresponds to first load and doesn't account for refetching stale data.